processor/resourcedetectionprocessor/README.md

```diff
@@ -40,6 +40,20 @@ to read resource information from the [GCE metadata server](https://cloud.google
     * host.name
     * host.type
 
+It also can optionally gather tags for the EC2 instance that the collector is running on
+
+EC2 custom configuration example:
+```yaml
+detectors: ["ec2"]
+ec2:
+    # A list of tags to add as resource attributes can be specified
+    tags_to_add:
```

Comment:
I wonder if we can use "*" for all the tags instead of `add_all_tags`?

Can we shorten `tags_to_add` to `tags`?